data/statuses.yaml

```diff
@@ -26,7 +26,7 @@
 - ident: released
   name: Dual support
   abbrev: "\u2714"
-  color: '7EBA54'
+  color: '5EBA84'
```

Comment:
This one feels particularly ugly to my eyes. But I understand that this is not objective feedback.

Reply:
Design changes do that. If it still looks jarring in a week, let's change it?